### ISSUE TITLE
Support downloading previous years certificates from student dashboard

### ISSUE THREAD

**Issue by dboyer** [Iridescent-CM/technovation-app#3921]
Depends on #3919

This issue covers the technical work to:

- [ ] figure out which years to display on a participant's dashboard

- [ ] display past certificates as downloadable files on the student dashboard

NOTE: if student does not have previous certificates, the tab remains visible with placeholder text indicating no previous certificates are available. 

<img width="669" alt="Screen Shot 2023-04-12 at 12 30 16 PM" src="https://user-images.githubusercontent.com/516764/231522540-1e275090-93fd-4eae-b9bb-0fdd0a4310b8.png">

**Comment by shaun-technovation** [comment]
The basic/rough draft functionality for this is on QA now, I can address feedback/additional work in another PR. 👍
